README.md

```diff
@@ -69,7 +69,7 @@ Enable enhanced completions and diagnostics by setting the `"fastEnable": true`
 
 You just need to setup VSCode to use your local typescript install as by default it will try and use a version of typescript it is bundled with.
 
-1. You need to create a `settings.json` file inside of a `.vscode` directory. We need to configure VSCode to see the locally installed typescript binary (ensure `typescript.tdsk` points to the `lib` directory of the project typescript install). You can see an example of this in this repository - `./example/.vscode/settings.json`.
+1. You need to create a `settings.json` file inside of a `.vscode` directory. We need to configure VSCode to see the locally installed typescript binary (ensure `typescript.tdsk` points to the `lib` directory of the project typescript install). You can see an example of this in this repository - `./example/.vscode/settings.json`. If npm has hoisted your typescript install, ensure the path you configure accounts for that.
```

Comment:
**You** need to create a `settings.json` file inside of a `.vscode` directory. **We** need to configure VSCode to see the locally
 
 Suggestion: I guess we could replace the "we" to something like "You'll need to configure"

Reply:
https://github.com/genesiscommunitysuccess/custom-elements-lsp/pull/52/commits/3b5026583508fe79c82b9d793ad05fd29fc13ca6